.github/workflows/ci.yml

```
@@ -4,6 +4,7 @@
 name: ci
 on:
   push:
+  pull_request:
```

this will lead to dupe executions

```suggestion
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```